tasks/process_agent.py

```diff
@@ -129,9 +130,12 @@ def build_dev_image(ctx, image=None, push=False, base_image="datadog/agent:lates
         ctx.run("cp tools/ebpf/Dockerfiles/Dockerfile-process-agent-dev {to}".format(to=docker_context + "/Dockerfile"))
 
         ctx.run("cp bin/process-agent/process-agent {to}".format(to=docker_context + "/process-agent"))
-
         ctx.run("cp bin/system-probe/system-probe {to}".format(to=docker_context + "/system-probe"))
-        ctx.run("cp bin/agent/agent {to}".format(to=docker_context + "/agent"))
+        if include_agent_binary:
+            ctx.run("cp bin/agent/agent {to}".format(to=docker_context + "/agent"))
+        else:
+            ctx.run("touch {tmp_dir}/tmp_file".format(tmp_dir=docker_context))
```

Comment:
Mind sticking a comment in here similar to the one in the Dockerfile explaining why this is needed?